Expecto.FsCheck/FsCheck.fs

```diff
@@ -44,46 +45,76 @@ module ExpectoFsCheck =
               |> FailedException |> raise
       }
 
-    let config =
-      { config with
-          Runner = runner }
+    let test config =
+      let config =
+        {
+          MaxTest = config.maxTest
+          MaxFail = 1000
+          Replay = Option.map Random.StdGen config.replay
+          Name = String.Empty
+          StartSize = config.startSize
+          EndSize = config.endSize
+          QuietOnSuccess = true
+          Every = fun _ _ -> String.Empty
+          EveryShrink = fun _ -> String.Empty
+          Arbitrary = config.arbitrary
+          Runner = runner
+        }
+      async {
+        Check.One(String.Empty, config, property)
+      }
+
+    let testCode =
+      match configs with
+      | None -> AsyncFsCheck (None, None, test)
+      | Some (testConfig, stressConfig) ->
+        AsyncFsCheck (Some testConfig, Some stressConfig, test)
+    TestLabel(name, TestCase (testCode, focusState), focusState)
 
-    let test = async {
-      Check.One(String.Empty, config, property)
-    }
+  /// Builds a test property with config
+  let testPropertyWithConfigs testConfig stressConfig name =
+    propertyTest "ftestPropertyWithConfig" Normal
+      (Some(testConfig,stressConfig)) name
 
-    TestLabel(name, TestCase (Async test, focusState), focusState)
+  /// Builds a test property with config that will be ignored by Expecto.
+  let ptestPropertyWithConfigs testConfig stressConfig name =
+    propertyTest "ftestPropertyWithConfig" Pending
+      (Some(testConfig,stressConfig)) name
+
+  /// Builds a test property with config that will make Expecto to
+  /// ignore other unfocused tests.
+  let ftestPropertyWithConfigs stdGen testConfig stressConfig name =
+    let testConfig = { testConfig with replay = Some stdGen }
+    let stressConfig = { stressConfig with replay = Some stdGen }
+    propertyTest "ftestPropertyWithConfig" Focused
+      (Some(testConfig,stressConfig)) name
 
   /// Builds a test property with config
   let testPropertyWithConfig config name =
-    propertyTest "ftestPropertyWithConfig" Normal config name
+    propertyTest "ftestPropertyWithConfig" Normal (Some(config,config)) name
```

Comment:
One is for normal tests and the other for stress testing. FsCheck e.g. EndSize may need to be larger for stress testing. If only one config is specified then it should be used for both.